libsolutil/StackTooDeepString.h

```diff
@@ -0,0 +1,28 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+#pragma once
+
+#include <string>
+
+namespace solidity::util
+{
+	static std::string stackTooDeepString =
+		"Stack too deep. "
+		"Try compiling with `--via-ir` (cli) or the equivalent `viaIR: true` (standard JSON). "
+		"Otherwise, try removing local variables.";
```

Comment:
This removes the information of how much deeper than allowed it actually is.
I'm fine with that, but consensus?

Reply:
I also think it's not such a useful information.
But I think we should suggest to also activate the optimizer, shouldn't we?

Reply:
Just added info about optimizer.